src/foam/nanos/dig/DigWebAgent.java

```diff
@@ -65,26 +64,20 @@ public void execute(X x) {
       pm.error(x, fe.getMessage());
     } catch (Throwable t) {
       logger.error(t);
-      DigUtil.outputException(x,
-          new GeneralException.Builder(x)
-            .setStatus(String.valueOf(HttpServletResponse.SC_INTERNAL_SERVER_ERROR))
-            .setMessage(t.getMessage())
-            .setMoreInfo(t.getClass().getName())
-            .build(),
-          format);
+      DigErrorMessage error = new GeneralException(t.getMessage());
+      error.setStatus(String.valueOf(HttpServletResponse.SC_INTERNAL_SERVER_ERROR));
+      error.setMoreInfo(t.getClass().getName());
+      DigUtil.outputException(x, error, format);
       pm.error(x, t.getMessage());
     } finally {
       pm.log(x);
     }
   }
 
   public void sendError(X x, int status, String message) {
-    DigUtil.outputException(x,
-      new GeneralException.Builder(x)
```

Comment:
uhoh .. looks like I'm not supposed to use the builder anymore for exceptions, is that right? i use it in some other places, so I can change those too if it has some unwanted side effects.

Reply:
Builder is fine, just not needed. It should be a last resort. Generates lots of garbage.